samples/simple_spellcheck.py

```diff
@@ -0,0 +1,55 @@
+# simple_spellcheck.py
+# Guy Dumais, 2018-08-30
+# Copyright (c) 2018 Element AI. All rights reserved.
+
+import os
+
+from spellchecker.symspell import SymSpell, Verbosity  # import the module
+
+
+def main():
+    # create object
+    initial_capacity = 83000
+    # maximum edit distance per dictionary precalculation
+    max_edit_distance_dictionary = 2
+    prefix_length = 7
+    sym_spell = SymSpell(initial_capacity, max_edit_distance_dictionary,
+                         prefix_length)
+    # load dictionary
+    dictionary_path = os.path.join(os.path.dirname(__file__), '../spellchecker',
```

Comment:
I suggest you use pkg_resources for finding files, it's a bit cleaner.

Example (I think it will work out of the box):

```
import pkg_resources

path = pkg_resources.resource_filename('spellchecker', 'frequency_dictionary_en_82_765.txt')
```

Reply:
The code was straight from the original project but I agree it needs improvements.